common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -487,6 +487,10 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
    * in the underlying Hadoop configuration.
    */
   public static enum ConfVars {
+    MSC_CACHE_ENABLED("hive.metastore.client.cache.enabled", true,
+            "This property enables a Caffeiene LoadingCache for Metastore client"),
+    MSC_CACHE_MAX_SIZE("hive.metastore.client.cache.maxSize", 1_000_000_000,
```

Comment:
Can you use a size validator?
```
"hive.metastore.client.cache.maxSize", "1Gb", new SizeValidator(),
```

Reply:
I will also do similar change in `MetatoreConf.java`